src/main/java/weavers/siltarae/mistake/service/FeedService.java

```diff
@@ -5,26 +5,39 @@
 import org.springframework.data.domain.Page;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
+import weavers.siltarae.like.domain.repository.LikeRepository;
 import weavers.siltarae.mistake.domain.Mistake;
 import weavers.siltarae.mistake.domain.repository.MistakeRepository;
 import weavers.siltarae.mistake.dto.request.FeedRequest;
 import weavers.siltarae.mistake.dto.response.FeedListResponse;
+import weavers.siltarae.mistake.dto.response.FeedResponse;
+
+import java.util.List;
+import java.util.stream.Collectors;
 
 @Service
 @RequiredArgsConstructor
 @Slf4j
 @Transactional
 public class FeedService {
     private final MistakeRepository mistakeRepository;
+    private final LikeRepository likeRepository;
 
-    public FeedListResponse getFeedList(FeedRequest request) {
+    public FeedListResponse getFeedList(FeedRequest request, Long memberId) {
         Page<Mistake> mistakes = switch (request.getFeedType()) {
             case FASTEST -> mistakeRepository.findByDeletedAtIsNullOrderByIdDesc(request.toPageable());
             case POPULAR -> mistakeRepository.findMistakesSortedByLikes(request.toPageable());
         };
 
-        return FeedListResponse.from(mistakes);
+        return FeedListResponse.from(getFeedResponseList(mistakes.getContent(), memberId), mistakes.getTotalElements());
```

Comment:
```suggestion
        List<FeedResponse> feedResponses = getFeedResponseList(mistakes.getContent(), memberId);
        return FeedListResponse.from(feedResponses, mistakes.getTotalElements());
```

이런 형태는 어떨까요? 한눈에 알아보기 더 쉬워질 것 같아요!